js/app.js

```diff
@@ -17,6 +17,10 @@ function bdaySort(contact1, contact2) {
 }
 
 function bdayAge(contact) {
+  if (!(contact.bday instanceof Date)) {
```

Comment:
When the contact data is transmitted to the alarm request, this one seems to perform a toString() method. I could use a pre-filter in the alarm success() so the bdayAge could be a pure function.

> Le 3 nov. 2015 à 21:39, Pierre Haessig <notifications@github.com> a écrit :
> 
> In js/app.js:
> 
> > @@ -17,6 +17,10 @@ function bdaySort(contact1, contact2) {
> >  }
> >  
> >  function bdayAge(contact) {
> > +  if (!(contact.bday instanceof Date)) {
> just out of curiosity, in what situation is this contact.bday not an instance of Date ?
> 
> —
> Reply to this email directly or view it on GitHub.
>